packages/services/src/processDPP.service.ts

```diff
@@ -49,7 +50,7 @@ export const processDPP: IService = async (data: any, context: IDppContext): Pro
     const linkResolverContext = context.dlr;
     const linkResolver = await registerLinkResolver(
       vcUrl,
-      dppContext.dlrIdentificationKeyType,
+      aiData.primary.ai as IdentificationKeyType,
```

Comment:
I think parsing the `aiData.primary.ai` to the `IdentificationKeyType` enum may no longer be accurate, given that the AI is now much more flexible.